parsers/jscript.c

```diff
@@ -2385,6 +2392,16 @@ static bool parsePrototype (tokenInfo *const name, tokenInfo *const token, state
 
 			token->scope = state->indexForName;
 
+			/*
+			 * We could make prototype_tokens a dynamic array,
+			 * but it doesn't seem worth the trouble.  How many
+			 * aliases does one need for one function?
+			 */
+			tokenInfo *const identifier_token = newToken ();
+			tokenInfo * prototype_tokens[] = { NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL };
```

Comment:
Putting just one NULL is enough. https://stackoverflow.com/questions/2589749/how-to-initialize-array-to-0-in-c 
```
tokenInfo * prototype_tokens[10] = { NULL };
```

Reply:
When thinking about performance, a raw C array on the stack is a good choice.
However, to make the code simple, I think ptrArray is better; you can forget about prototype_token_count if you use ptrArray.

Reply:
Switched to ptrArray